chacra/models/binaries.py

```diff
@@ -28,6 +29,9 @@ class Binary(Base):
     project_id = Column(Integer, ForeignKey('projects.id'))
     project = relationship('Project', backref=backref('binaries', lazy='dynamic'))
 
+    repo_id = Column(Integer, ForeignKey('repos.id'))
```

Comment:
This was discussed in IRC, the m2m relationships will be configured instead of having a db relationship.